src/modules/auth/state.js

```diff
@@ -5,20 +5,35 @@ import {push} from 'connected-react-router';
 const REGISTER_REQ = 'auth/REGISTER_REQ';
 const REGISTER_REQ_SUCCESS = 'auth/REGISTER_REQ_SUCCESS';
 const REGISTER_REQ_FAIL = 'auth/REGISTER_REQ_FAIL';
+const LOGIN_REQ = 'auth/LOGIN_REQ';
+const LOGIN_REQ_SUCCESS = 'auth/LOGIN_REQ_SUCCESS';
+const LOGIN_REQ_FAIL = 'auth/LOGIN_REQ_FAIL';
+const AUTHENTICATE_REQ = 'auth/AUTHENTICATE_REQ';
+const AUTHENTICATE_REQ_SUCCESS = 'auth/AUTHENTICATE_REQ_SUCCESS';
+const AUTHENTICATE_REQ_FAIL = 'auth/AUTHENTICATE_REQ_FAIL';
```

Comment:
@rtluu @chasdevs We should have a talk sometime about how we want to handle creating and managing our Redux actions.

Redux is so aggressively verbose that we need a way to streamline the creation of actions and reducers.

The simplest way is to create a boilerplate generator so we can just `generate-redux-action login` and it will create the file and manage the imports.

A more complex but more elegant way is to actually create a system where whenever we create a new initialState, a number of default actions get created automatically (setters, success/fail etc), and also a bit of boiler plate is generated via command line tools. This is what I do on other projects, but it is more complicated.

We should talk

Reply:
Ooo interesting, yea we should connect. Are you both free tonight? Or maybe tomorrow?

Reply:
Not tonight but maybe tomorrow if it's early my time